codebundles/sli-incident-threshold/README.md

```diff
@@ -0,0 +1,36 @@
+# SLI Incident Threshold
+This codebundle allows you to monitor another SLI and trigger a TaskSet when the expected rate of a SLI value falls below a specified threshold.
+
+## SLI
+Depending on your observability needs, the Multi-Window Multi-Burn algorithm + SLO error budgets approach may not apply to your use case. In those cases you can use this codebundle to create an incident threshold based on another SLI. A query will be performed on the monitored SLI's metrics for a given time window and resolution, and then the presence of a success value will be checked. For example: fetch 1 hour of metric data at 5 minute intervals, for the monitored SLI; a `0` means failure and `1` means healthy. If we set the success value to `1` and a rate of `1.0` (100%) then when any failure occurs in the monitored SLI, this codebundle will immediately trigger the given TaskSet.
+
+### Use Case: SLI: Trigger a slack message when my API health check fails
+For our public API, it's uptime is critical, so we can monitor its health check and send a slack message to a team channel whenever the health check fails.
+
+```
+configProvided:
+  - name: WORKSPACE_NAME
```

Comment:
Originally I wrote the codebundle with the REST keywords to allow this but there were concerns over exposing too much of the public api flow while things are still in flux and subject to change (particularly dealing with long lived rest tokens). I rewrote it as a built-in papi keyword. The intent is to allow users to monitor SLIs outside their workspace if desired. Apparently this can be done by adding the workspace service account to the workspace rbac settings but I've not tried this yet.

Reply:
so we are leaving the variable in place here, but currently it only works "in workspace" until we've tested out rbac settings?

Reply:
I was thinking yes, since it encourages thinking along the lines of "Oh I could do this..." when someone is reading about it. In terms of hooking it up to other workspaces (like the public GCP workspace) once we've got that